File: api/api/Models/Journal.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models
{
    [Table("Journals")]
    public class Journal
    {
        public int TripId { get; set; }
        public Trip Trip { get; set; }
        public int JournalId { get; set; }
        public string Title { get; set; }
        public string Entry { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public string AppUserId { get; set; }
        public AppUser AppUser { get; set; }
    }
}

File: frontend/src/Models/Journal.ts
export type JournalGet = {
    tripId: number;
    journalId: number;
    title: string;
    entry: string;
    createdAt: string;
    createdBy: string;
}

export type JournalPost = {
    tripId: number;
    title: string;
    entry: string;
}

export type JournalDelete = {
    id: number;
}